src/template_partials/templatetags/partials.py

```diff
@@ -56,33 +58,43 @@ def render(self, context):
         nodelist = self.origin.partial_contents[self.partial_name]
         return nodelist.render(context)
 
-
-@register.tag(name="startpartial")
-def startpartial_func(parser, token):
+@register.tag(name="partialdef")
+def partialdef_func(parser, token):
     """
     Declare a partial that can be used later in the template.
 
     Usage:
 
-        {% startpartial "partial_name" %}
+        {% partialdef "partial_name" %}
 
         Partial content goes here
 
-        {% endpartial %}
+        {% endpartialdef %}
 
     Stores the nodelist in the context under the key "partial_contents" and can
     be retrieved using the {% partial %} tag.
     """
-    # Parse the tag
+    _define_partial(parser, token, "endpartialdef")
+    return DefinePartialNode
```

Comment:
Surely... 

```suggestion
    return _define_partial(parser, token, "endpartialdef")
```

... no? 🤔

Reply:
Yup, don't know how I made that mistake :) 

I fixed the merge conflicts as well (with the new inline option).  Hope it's all good and merge-able now!

Reply:
Thanks for the update @scriptogre — I will take another look.